TFT/src/User/Menu/Heat.c

```diff
@@ -276,7 +276,7 @@ void loopCheckHeater(void)
   {  /* Send M105 query temperature continuously	*/
     if(update_waiting == true)                {nowTime=OS_GetTime();break;}
     if(OS_GetTime()<nowTime+update_time)       break;
-
+    if(RequestCommandInfoIsRunning())          break; //to avoid colision in Gcode response processing

```

Comment:
This looks same as `if(update_waiting == true)` in `line no: 277` which is also used to avoid sending gcode commands when there are already some commands waiting for response. Please check and make sure no duplicate codes exists.

Reply:
For my understanding update_waiting is a flag used to know if we are waiting for the next occurrence of temperature update  ,it is an  internal management flag  so  not duplicate with my  condition that will delay the update until we have finish with  to retrieve file list or long filename